experimental/dds/tree/src/publicContracts.ts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IChannelAttributes } from '@fluidframework/datastore-definitions/internal';
+
+/**
+ * {@inheritDoc @fluidframework/shared-object-base#ISharedObjectFactory."type"}
+ * @alpha
+ */
+export const SharedTreeType: string = 'SharedTree';
```

Comment:
```suggestion
export const SharedTreeType = 'SharedTree';
```

this will make the type of the variable be `'SharedTree'` rather than `string`